### PR TITLE
Huvud Chain Mount

### PR DESCRIPTION
**Pull request by bdbell**
* [ ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ ] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ ] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [ ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [ ] Submitted STL files are oriented and scaled properly for printing.
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

This is a mod to allow mounting the Huvud toolhead board to the rear of a Clockwork or LGX extruder and still have use of the spec cable chains.

#### How was it tested? 

Printed and installed on my own printer.  See pictures in README.

#### Any background context you want to provide?

Normally installing a Huvud forces the use of an umbilical because there is nowhere to mount the X chain to.  This solves that problem.  positioning the Huvud in the standard position with the bolt patterns aligned causes the lower portion of the chain and the Huvud to interfere.  This took 3 revisions to get right and to simplify.

#### Screenshots (if appropriate)

See README and images.

### CODE REVIEW

File: printer_mods/120decibell/huvud_chain_mount/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+* (3) M3x6 FHCS to mount the chain
+* (2) M3x25 BHCS to mount to the motor
+* (4) M3x8 SHCS to mount the Huvud.
+
+Installation:
+
+* Install the 3 heat set inserts.
+* Remove the upper two screws of the motor.
+* Install the mount with the two M3x25 BHCS.
+* Attached the Huvud with the four M3x8 SHCS.  The screws will thread into the plastic. _Be sure to check clearance between electrical components and the M3 BHCS screws underneath_
```
Comment (KiloQubit) [inline printer_mods/120decibell/huvud_chain_mount/README.md:17]: Replace `Attached` with `Attach`.

File: printer_mods/120decibell/huvud_chain_mount/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Parts Required:
+
+* (3) M3 heat set inserts for the chain
+* (3) M3x6 FHCS to mount the chain
+* (2) M3x25 BHCS to mount to the motor
+* (4) M3x8 SHCS to mount the Huvud.
+
+Installation:
+
```
Comment (KiloQubit) [inline printer_mods/120decibell/huvud_chain_mount/README.md:13]: It would be good to mention that the user should remove the built-in support.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!  Minor readme changes below.

**Comment by bdbell** [comment]
Updates made as requested

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**